<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtMqttClass</class>
 <widget class="QMainWindow" name="QtMqttClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtMqtt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout" columnstretch="3,1">
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="SubsLabel">
          <property name="text">
           <string>当前订阅</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="subsList"/>
        </item>
        <item>
         <widget class="QLineEdit" name="subNameInput">
          <property name="inputMask">
           <string/>
          </property>
          <property name="placeholderText">
           <string>输入订阅名称</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="newSubsButton">
            <property name="text">
             <string>新增订阅</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="unsubscribeButton">
            <property name="text">
             <string>退订</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearSubsButton">
            <property name="text">
             <string>清除所有订阅</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="receivedMessageLabel">
          <property name="text">
           <string>日志</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="messageList"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="startButton">
            <property name="text">
             <string>新的连接...</string>
            </property>
            <property name="default">
             <bool>true</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stopButton">
            <property name="text">
             <string>断开连接</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearButton">
            <property name="text">
             <string>清空</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>867</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_Help"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>关于...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtMqtt.qrc"/>
 </resources>
 <connections/>
</ui>
